packages/cli/src/lib/project/dotFolder.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  createFileSync,
+  existsSync,
+  pathExistsSync,
+  writeJSONSync,
+} from 'fs-extra';
+import {join} from 'path';
+
+export class DotFolder {
+  public constructor(public owner: string) {
+    this.ensureFolderExists();
+  }
+
+  public static get hiddenFolderName() {
+    return '.coveo';
+  }
+
+  public get path() {
+    return join(this.owner.toString(), DotFolder.hiddenFolderName);
+  }
+
+  private ensureFolderExists() {
```

Comment:
If you are using fs-extra, can you just use [ensureDir](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/ensureDir.md)?